compiler/circle-eval-diff/driver/Driver.cpp

```diff
@@ -14,12 +14,38 @@
  * limitations under the License.
  */
 
+#include "CircleEvalDiff.h"
+
 #include <arser/arser.h>
 #include <vconone/vconone.h>
 
+using namespace circle_eval_diff;
+
 namespace
 {
 
+std::string to_lower_case(std::string s)
+{
+  std::transform(s.begin(), s.end(), s.begin(), [](unsigned char c) { return std::tolower(c); });
+  return s;
+}
+
+Metric str_to_metric(const std::string &str)
```

Comment:
How about rename it to `to_metric` like other functions? `str` keyword in function name seems not needed.

Reply:
Ah, I made this draft before hearing your naming seminar and forgot to update it.